assets/styles/authoring/layout/workspace.scss

```diff
@@ -25,9 +25,14 @@
 
   .workspace-header {
     width: 100%;
-    padding: 12px 47px 6px 47px;
+    padding: 6px 10px 3px 20px;
```

Comment:
With this padding change, the live view loading indicator now lays on top of the account icon

![loading](https://user-images.githubusercontent.com/6248894/125625888-12b2c620-90e0-4d95-851e-e0a152ee5ec5.gif)